linux_os/guide/services/ssh/ssh_server/sshd_enable_gssapi_auth/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
Please remove `# profiles` line from all 3 tests. This rule isn't a part of any profile. And the test suite won't run the test scenarios unless the user explicitly overrides this setting.

Reply:
Can do, I'll have a fix up shortly.